cargo-ledger/src/setup.rs

```diff
@@ -39,4 +39,42 @@ pub fn install_targets() {
             println!("* {target} already installed");
         }
     }
+
```

Comment:
For those who already installed targets, `cargo ledger setup` won't be able to tell the difference between the old targets and the new ones and wont install the .json present in this patch. Should we handle that or just explain that the targets should be manually removed beforehand?

Reply:
Ok we should handle this yes. The most simple could be to overwrite systematically the targets even if they are existing ?